scaffold/snabbdom/webpack.config.js

```diff
@@ -12,14 +37,11 @@ module.exports = {
     loaders: [{
       test: /\.js?$/,
       exclude: /node_modules/,
-      loader: 'babel-loader',
+      loader: 'babel',
       query: {
-        "presets": ["es2015", "stage-0"],
-        "plugins": []
+        presets: ['react', 'es2015', 'stage-0']
```

Comment:
@bfitch 
I think it is copy paste error. inferno and none also has wrong config